### DISCUSSION TITLE
On tty it loops ,why ?

### DISCUSSION THREAD

**Discussion by rexackermann** [mcdope/pam_usb#198] [Q&A] [answered]
Okay ,so on a terminal emulator it's working as expected but when i try to login on tty ,it just loops to ask for username again.
Can anyone tell me why it's happening ?

There is a disturbing messege,

libdisks2-CRITICAL: 10/251.666: udisks_client_get_object manager: assertion 'UDISKS IS CLIENT (client)' failed

**Comment by mcdope** [comment, marked as answer]
Continued in #199, so closing this discussion.
